holidays/financial/six_swiss_exchange.py

```
@@ -0,0 +1,67 @@
+        [2025](https://web.archive.org/web/20250908065713/https://www.six-group.com/dam/download/the-swiss-stock-exchange/trading/trading-provisions/regulation/trading-guides/trading-calendar-2025.pdf)
+        [2026](https://web.archive.org/web/20251225182641/https://www.six-group.com/dam/download/the-swiss-stock-exchange/trading/trading-provisions/regulation/trading-guides/trading-calendar-2026.pdf)
+    """
+
+    country = None  # type: ignore[assignment]
+    market = "XSWX"
+    parent_entity = Switzerland
+    parent_entity_subdivision_code = "ZH"
+    supported_categories: tuple[str] = (PUBLIC,)
+    start_year = 2000
+
+    def _add_holiday(self, name: str, *args) -> date | None:
```

Ruff (0.15.21)

[warning] Missing type annotation for `*args`

(ANN002)